src/views/GifReferenceWidget.vue

```diff
@@ -22,38 +22,57 @@
 <template>
 	<div class="gif-reference">
 		<div class="gif-wrapper">
-			<div v-if="!isLoaded" class="loading-icon">
+			<div v-if="!isLoaded && gifsEnabled" class="loading-icon">
 				<NcLoadingIcon
 					:size="44"
 					:title="t('integration_giphy', 'Loading GIF')" />
 			</div>
-			<img v-show="isLoaded"
-				class="image"
-				:src="proxiedUrl"
-				@load="isLoaded = true">
-			<a v-show="isLoaded"
-				class="attribution"
-				target="_blank"
-				:title="poweredByTitle"
-				href="https://giphy.com">
-				<img :src="poweredByImgSrc"
-					:alt="poweredByTitle">
-			</a>
+			<NcButton :pressed.sync="gifsEnabled"
+				class="toggle-gifs-button"
+				:type="gifsEnabled ? 'secondary' : 'primary'"
+				:aria-label="ariaLabel"
+				:title="ariaLabel"
```

Comment:
```suggestion
				:title="hideButtonTitle"
```
No need for an aria-label if a title is set.